mobile-app/src/components/habit-components/HabitCard.tsx

```diff
@@ -11,7 +11,13 @@ import HabitCompletionsWeeklyView from "./HabitCompletionsWeeklyView";
 import { HabitFriendCompletions } from "./HabitFriendCompletions";
 import { HabitHeader } from "./HabitHeader";
 
-export function HabitCard({ habitId }: { habitId: string }) {
+export function HabitCard({
+  habitId,
+  screenType = "",
+}: {
+  habitId: string;
+  screenType?: "" | "/modals";
+}) {
```

Comment:
This seems confusing to me. How about instead, you have an openAsModal argument that defaults to false?